lab_1/main.py

```diff
@@ -13,27 +13,58 @@ def tokenize(text: str) -> list or None:
     """
     pass
 
+    import re   # импортируем модуль re для последующей работы с регулярными выражениями
```

Comment:
`import` statements are (usually) place at the beginning of the file - after the documentation string.